### ISSUE TITLE
importlib metadata failure with Python 3.7 and 3.8

### ISSUE THREAD

**Issue by Lawouach** [chaostoolkit/chaostoolkit#279]
```
$ chaos --version
Traceback (most recent call last):
  File "/.venv/bin/chaos", line 5, in <module>
    from chaostoolkit.__main__ import cli
  File "/.venv/lib/python3.7/site-packages/chaostoolkit/__main__.py", line 1, in <module>
    from chaostoolkit.cli import cli
  File "/.venv/lib/python3.7/site-packages/chaostoolkit/cli.py", line 762, in <module>
    with_plugins(importlib_metadata.entry_points().get("chaostoolkit.cli_plugins"))(cli)
AttributeError: 'EntryPoints' object has no attribute 'get'
```

**Comment by dmartin35** [comment]
Make sure to pin 'importlib-metadata' to version < 5. 
They removed deprecated entry point interfaces. 

I had same issue a couple of weeks ago ;)

**Comment by Lawouach** [comment]
Thanks David. I did that and I also tried to be more tolerant about the actual version that is installed when I call it. It's unfortunate but I hope this'll work.

**Comment by Lawouach** [comment]
Released in 1.15.1
